resources/js/Components/Toast/Toast.tsx

```diff
@@ -39,7 +39,7 @@ export const Toast = forwardRef<HTMLDivElement, ToastProperties>(
                         "bg-theme-success-100 text-theme-success-700": type === "success",
                         "bg-theme-warning-100 text-theme-warning-800": type === "warning",
                         "bg-theme-danger-100 text-theme-danger-700": type === "error",
-                        "bg-theme-hint-100 text-theme-hint-700": type === "info",
+                        "bg-theme-primary-100 text-theme-primary-700": type === "info",
```

Comment:
we should keep it as `hint` in the toasts, as these elements are purple regardless of the primary theme. For all other instances I agree that changing it to primary makes more sense